### PR TITLE
Feature/user

### LINKED ISSUE #2: display user info
As a user, I should see my info being displayed upon page loading

- [ ] profile section(name, img, rating, hours)
- [ ] upcoming job section

### LINKED ISSUE #3: Style user profile
Update styles for user profile section

### PR DESCRIPTION
**Pull request by coopterrones**
#### What's this PR do?
1. Creates / cleans up file architecture
2. add `apiCalls` helper functions into `apiCalls.js`
3. request user info using apiCalls
4. display user information in user sidebar from network request
5. display upcoming jobs with conditional render
6. styling updates to user profile sidebar
7. add star images to user rating section

#### Where should the reviewer start?
-`App.js` -- creates functional component for app -- line 7
-`App.js` -- getUserInfo using apiCalls -- line 10
-`App.js` -- useEffect -- line 17
-`App.js` -- conditional rendering -- line 19 
-`User.scss` updates for all styling in user view 
-`User.js` -- creates user functional component - line 1 
-`User.js` -- all information added for user profile in sidebar 
-`apiCalls.js` -- all getUser helper function created - line 1 

#### How should this be manually tested?
- `npm start` -- check that all user info displays correctly and no bugs... 

#### Screenshots (if appropriate):
Screen Shot 2021-01-05 at 8 13 38 PM

### CODE REVIEW

**Review by yiranli624 — APPROVED** [review]
Great work! Ready for some testing 👍

File: src/Components/User/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <div className="hours-bar">
+        <p>{volunteeredHours} Hours</p>
+      </div>
+      <div className="section-titles">
+        <p>My Upcoming Jobs</p>
+        <hr className="section-line" />
+      </div>
+      {upcomingJobs.map((job) => {
+        return (
+          <section key={job.id} className="upcoming-job-card">
```
Comment (yiranli624) [inline src/Components/User/index.js:45]: Could potentially break it into multiple components

**Review by yiranli624 — APPROVED** [review]

**Review by okalimbet — COMMENTED** [review]
Good apiCalls organization. Thank you for separating it from the rest of the files

File: src/Components/User/index.js [hunk 2]
```diff
@@ -10,28 +10,37 @@ const User = ({ info }) => {
...
   return (
-    <section id={id}>
-      <div className="profile-picture">
-        <img src={profilePicture} />
-      </div>
+    <section className="user-wrapper" id={id}>
+      <div
+        className="profile-picture"
+        style={{
+          backgroundImage: `url(${profilePicture})`,
```
Comment (okalimbet) [inline src/Components/User/index.js:18]: good use of dynamic background image style!

**Review by okalimbet — APPROVED** [review]

File: src/Components/User/index.js [hunk 3]
```diff
@@ -21,7 +22,13 @@ const User = ({ info }) => {
         }}
       ></div>
       <h3 className="username">{name}</h3>
-      <div className="rating-wrapper">{rating}</div>
+      <div className="rating-wrapper">
+        <img className="star-image" src={starImage}/>
```
Comment (okalimbet) [inline src/Components/User/index.js:26]: Nice work with star images! :D

**Review by okalimbet — APPROVED** [review]

File: src/Components/User/User.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+.hours-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: $main-white;
+  background-color: $main-blue;
+  height: 2em;
+  width: 50%;
+  margin: 0.5em auto;
```
Comment (okalimbet) [inline src/Components/User/User.scss:41]: We need to create more variables or mixins to avoid repetitiveness like margin: 0.5em

**Review by okalimbet — APPROVED** [review]

**Pull request merged**